components/eam/docs/user-guide/aerosol_phys_prop.md

```diff
@@ -0,0 +1,111 @@
+
+# Aerosol physical properties
+
+## Description
+
+Key information
+
+- Original physical properties of aerosols are documented in Liu et al. (2012) [@liu_toward_2012]. Detailed information is included in the supplement.
+
+- Physical properties of aerosols used in E3SMv1 are documented in Wang et al. (2020) [@wang_aerosols_2020].
+
```

Comment:
Physical properties of the fifth mode aerosols are documented by Ke et al. (2024, in preparation)